### PR TITLE
hosting-provider: add a quick warning

### PR DESCRIPTION
**Pull request by Mikaela**
## Description

Resolves: #1341

* Netlify preview for the mainly edited page: https://deploy-preview-1464--privacytools-io.netlify.com/providers/hosting/

### CODE REVIEW

**Comment by Mikaela** [comment]
Linda:

> https://github.com/privacytoolsIO/privacytools.io/pull/1464
> The problem at https://github.com/privacytoolsIO/privacytools.io/issues/1341 is more about "won't track your visitors or give into government data requests" being dangerous to say. The quick warning is "fine", but it's not fine because... it's self-explanatory and not really a warning at all.
> It's best to question why such warning would be there in first place, as if PTIO didn't know what it was recommending or why.
> And honestly, I *don't* have an idea why PTIO is recommending any of those but Njalla.
> I know Bahnhof, but I don't know why it's on that list.
> Companies also have to keep records for tax purposes, etc.

**Comment by Mikaela** [comment]
I don't know these providers well personally, but I would argue that a warning is necessary as we aren't all-knowing.

See also https://github.com/privacytoolsIO/privacytools.io/issues/1196

File: _includes/sections/hosting-provider.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <h1 id="host" class="anchor"><a href="#host"><i class="fas fa-link anchor-icon"></i></a> Secure Hosting Provider</h1>
 
+<div class="alert alert-warning" role="alert">
+  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it.</strong>
```
Comment (nitrohorse) [inline _includes/sections/hosting-provider.html]: What about adding an additional suggestion to verify their privacy policies?
```suggestion
  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please verify their privacy policies.</strong>
```

Comment (Mikaela) [reply _includes/sections/hosting-provider.html]: I committed this, but would "read" make more sense than "verify" ?

Comment (nitrohorse) [reply _includes/sections/hosting-provider.html]: I meant verifying their claims on their main home pages, but yeah “read” probably is more clear.

File: _includes/sections/hosting-provider.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <h1 id="host" class="anchor"><a href="#host"><i class="fas fa-link anchor-icon"></i></a> Secure Hosting Provider</h1>
 
+<div class="alert alert-warning" role="alert">
+  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please verify their privacy policies.</strong>
```
Comment (Mikaela) [inline _includes/sections/hosting-provider.html]:
```suggestion
  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please read their privacy policies.</strong>
```

Lazy suggestion, so I don't have to terminal it.

File: _includes/sections/hosting-provider.html (hunk 1 above)
Comment (Mikaela) [reply _includes/sections/hosting-provider.html]: Changed to read, please rereview

**Comment by JonahAragon** [comment]
I do not think this is necessary, or we would have to add such a warning to every page on the site. VPN and DNS providers seem like special cases because they will receive your data when you are visiting third parties. Hosting providers will only see your data when you are directly accessing their servers.

If we want to add a generic warning, adding some text for clarification of our point and putting it on https://www.privacytools.io/providers/ would be a better solution.

**Comment by Mikaela** [comment]
> Hosting providers will only see your data when you are directly accessing their servers. 

I don't understand how to interpret this. Hosting provider has root on your servers and thus see my data when I am using anything not-E2EE on a server hosted on them.

**Comment by Mikaela** [comment]
I am unlikely to work on this any time soon.

**Pull request closed without merging**